### PR TITLE
feat(legacy): limit `tui-tag` width to `80%` to avoid wrapping in selects

### LINKED ISSUE #4255: 🐞 - Empty line in multi-select in case of long tags
### Playground Link

https://stackblitz.com/edit/angular-ycryww?file=src%2Fapp%2Fapp.component.ts

### Description

When tag values in multi-select are very long, there is an empty line (even if select not focused).

### Angular version

12.0.0

### Taiga UI version

2.83.0

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by splincode**
Fixes #4255

### Before

image

### After

image

### CODE REVIEW

File: projects/legacy/components/input-tag/input-tag.style.less [hunk 1]
```diff
@@ -82,6 +82,10 @@
     :host[data-size='l']:not(._label-outside) & {
         padding: 0;
     }
+
+    & > tui-tag:nth-child(1):has(+ .t-input-wrapper) {
```
Comment (waterplea) [inline projects/legacy/components/input-tag/input-tag.style.less:86]: Good idea, but it breaks if there are tags before:
image
How about we just make :last-child tag max width 80%?

Comment (splincode) [reply projects/legacy/components/input-tag/input-tag.style.less]: @waterplea fixed

File: projects/legacy/components/input-tag/input-tag.style.less [hunk 2]
```diff
@@ -82,6 +82,10 @@
     :host[data-size='l']:not(._label-outside) & {
         padding: 0;
     }
+
+    & > tui-tag:has(+ .t-input-wrapper) {
```
Comment (waterplea) [inline projects/legacy/components/input-tag/input-tag.style.less:86]: No need for `:has` stuff, just the `:last-child` — we don't want a single tag to only take up 80% of the input if it is alone on the row, we only want the last one which is right next to input

Comment (splincode) [reply projects/legacy/components/input-tag/input-tag.style.less:86]: I don't understand how it should work
image

image

Comment (waterplea) [reply projects/legacy/components/input-tag/input-tag.style.less:86]: It's not `:last-child`, apparently:
image
`:last-of-type` should work.

Comment (splincode) [reply projects/legacy/components/input-tag/input-tag.style.less]: fixed

**Review by waterplea — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/demo/src/modules/components/multi-select/examples/1/index.ts [hunk 3]
```diff
@@ -9,7 +9,7 @@ import {TuiDataListWrapper} from '@taiga-ui/kit';
 import {TuiMultiSelectModule, TuiTextfieldControllerModule} from '@taiga-ui/legacy';
 
 const ITEMS: readonly string[] = [
-    'Luke Skywalker',
+    'Luke Skywalker and a long time ago in a galaxy far, far away..',
```
Comment (vladimirpotekhin) [inline projects/demo/src/modules/components/multi-select/examples/1/index.ts:12]: nit: looks like one example of a long tag is enough for the test

Comment (splincode) [reply projects/demo/src/modules/components/multi-select/examples/1/index.ts:12]: fixed

**Pull request merged**